packages/core/src/fhir-to-cda/fhir.ts

```diff
@@ -16,6 +17,10 @@ function isOrganization(resource: Resource | undefined): resource is Organizatio
   return resource?.resourceType === "Organization";
 }
 
+export function isCondition(resource: Resource | undefined): resource is Condition {
+  return resource?.resourceType === "Condition";
+}
+
 export function isComposition(resource: Resource | undefined): resource is Composition {
   return resource?.resourceType === "Composition";
 }
```

Comment:
ditto: have this on the FHIR folder?

Reply:
Yea, will do so on a follow up PR

Reply:
Why a follow-up PR? Is this a patch resolving an issue impacting cxs? If not, and not a lot more work, it would be better to at least have the new ones already on the desired place, so we have less stuff to update/move afterwards and can focus on the next thing.

Reply:
I thought that's what you meant on another comment. If not, I can move this whole file there now

Reply:
If moving existing code adds a lot to this PR, no worries w/ doing that on a follow-up PR. I mostly referring to avoid creating stuff that we now needs update later. But just a general guideline, I'm trying to keep a balance between having the code easier to maintain and to our work queue manageable. 😄

Reply:
Gotcha. Moved!